video/out/w32_common.c

```diff
@@ -963,6 +963,21 @@ static bool is_visible(HWND window)
     return GetWindowLongPtrW(window, GWL_STYLE) & WS_VISIBLE;
 }
 
+//Set the mpv window's affinity.
+//This will affect how it's displayed on the desktop and in system-level operations like taking screenshots.
+//During startup of mpv we want to avoid changing anything when its not needed, that is toggled by handleDefault
+static bool update_affinity(struct vo_w32_state *w32, bool handleDefault)
+{
+    bool affinity_result = false;
+    if (!w32 || w32->parent) {
+        return affinity_result;
+    }
+    if (w32->opts->window_affinity != WDA_NONE || handleDefault) {
+        affinity_result = SetWindowDisplayAffinity(w32->window, w32->opts->window_affinity);
```

Comment:
Can't you just pass `WDA_NONE` to undo/not set the window affinity? Or does it not work like that.

Reply:
What do you mean?

Reply:
What I mean is why are you avoiding `SetWindowDisplayAffinity` if the option value is `WDA_NONE`? Shouldn't that just seamlessly work? It also makes it impossible to unset window affinity.

Reply:
Assuming, WDA_NONE is 0 (not sure if it is or not), you can just do `if (w32->opts->window_affinity)` in the startup code. And probably get rid of the whole `handleDefault` logic in ` update_affinity` while you're at it.

Reply:
You are correct, thanks for the tip, I've changed up that logic now and it just has the conditional on start up and the main function is simplified